include/tests_time

```diff
@@ -574,7 +574,7 @@
 
 
     Register --test-no TIME-3185 --preqs-met "${PREQS_MET}" --weight L --network NO --category "security" --description "Check systemd-timesyncd synchronized time"
-    SYNCHRONIZED_FILE="/run/systemd/timesync/synchronized"
+    SYNCHRONIZED_FILE="/var/lib/systemd/timesync/clock"
```

Comment:
Maybe a if read test on `/run/systemd/timesync/synchronized` so we catch older versions as well?

Reply:
@konstruktoid `/var/lib/systemd/timesync/clock` exists in both old and new versions, but `/run/systemd/timesync/synchronized` only exists in new version (unfortunately I did not find an entry in the systemd changelog to tell which version it is), but I agree it would be better to test multiple files.

The systemd changelog also mentions ` /var/lib/private/systemd/timesync/clock`, but I am not sure under which circumstances this file exists: https://github.com/systemd/systemd/blob/master/NEWS#L2034-L2043. If I interpret it correctly, the file exists in `/var/lib/private` if DynamicUser option is used.

Reply:
You're correct, setting `DynamicUser=yes` in `/lib/systemd/system/systemd-timesyncd.service` creates `/var/lib/private/systemd/timesync/clock`.

```
$ sudo stat -c "%n: %z" /var/lib/systemd/timesync/clock /run/systemd/timesync/synchronized /var/lib/private/systemd/timesync/clock
/var/lib/systemd/timesync/clock: 2020-08-24 09:41:52.030011075 +0000
/run/systemd/timesync/synchronized: 2020-08-24 09:41:52.030011075 +0000
/var/lib/private/systemd/timesync/clock: 2020-08-24 09:41:52.030011075 +0000
$ systemd --version && uname -r && lsb_release -a
systemd 245 (245.4-4ubuntu3.2)
+PAM +AUDIT +SELINUX +IMA +APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD +IDN2 -IDN +PCRE2 default-hierarchy=hybrid
5.4.0-42-generic
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.1 LTS
Release:        20.04
Codename:       focal
```

Reply:
@konstruktoid I added a check for both `/var/lib/systemd/timesync/clock` and `/var/lib/private/systemd/timesync/clock`